demo/Cimpress.Nancy.Demo/DemoNancyServiceBootstrapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cimpress.Nancy;
+using Cimpress.Nancy.Components;
+using Nancy.Bootstrapper;
+using Nancy.TinyIoc;
+
+namespace Cimpress.Nancy.Demo
+{
+    public class DemoNancyServiceBootstrapper : NancyServiceBootstrapper
+    {
+        public DemoNancyServiceBootstrapper() : base()
+        {
+            
+        }
+
+        protected override void ApplicationStartup(TinyIoCContainer container, IPipelines pipelines)
+        {
+            container.Register<INancyLogger, DemoLogger>();

```

Comment:
Do you need this? 
If I recall, the register with default function will register whatever class implements INancyLogger for you.